app/Dockerfile

```diff
@@ -41,7 +41,11 @@ RUN sed -i -nr '/node|yarn/p' ${APP_HOME}/.tool-versions && \
   cat ${APP_HOME}/.tool-versions | cut -f 1 -d ' ' | xargs -n 1 asdf plugin-add && \
   asdf plugin-update --all && \
   asdf install && \
-  asdf reshim
+  asdf reshim && \
+  pushd ${APP_HOME}/.asdf/installs/nodejs/$(awk '/^nodejs/ { print $2 }' .tool-versions)/lib && \
+  npm i npm corepack && \
+  rm -f package.json package-lock.json && \
+  popd
```

Comment:
Is this intentionally in this PR?

Reply:
yes I rebased it on the main I already merged but haven't pushed yet because I'm waiting for happo to pass (don't judge)

Reply:
it'll disappear on its own

Reply:
now it's gone 🪄

Reply:
![magic](https://media.giphy.com/media/Ri7d8I18cto2jufOKc/giphy.gif)